docs/curl-commands.md

```diff
@@ -0,0 +1,33 @@
+# Customs Declarations Curl Commmands
+---
+### Endpoints Summary
+
+| Path                                          | Method | Description                                |
+|-----------------------------------------------|--------|--------------------------------------------|
+| [`/`](#user-content-post-export-payload)      | `POST` | Allows submission of an inventory linking export payload |
+
+#### Useful CURL commands for local testing
+[link to curl commands](docs/curl-commands.md)
+--- 
+ 
+### POST export payload 
+#### `POST /`
+Allows submission of an inventory linking export payload
+ 
+##### curl command
+```
+curl -v -X POST "http://localhost:9823/" \
+  -H 'Accept: application/vnd.hmrc.1.0+xml' \
+  -H 'Authorization: Bearer {ADD VALID TOKEN}' \
```

Comment:
for consistency with other header values below and in imports consider just having `token` instead of `ADD VALID TOKEN`

Reply:
done